src/fides/api/util/rate_limit.py

```diff
@@ -180,15 +180,32 @@ async def dispatch(self, request: Request, call_next):  # type: ignore
     CONFIG.security.rate_limit_client_ip_header is not None
     and CONFIG.security.rate_limit_client_ip_header != ""
 )
-fides_limiter = Limiter(
-    storage_uri=CONFIG.redis.connection_url_unencoded,
-    application_limits=[
-        CONFIG.security.request_rate_limit
-    ],  # Creates ONE shared bucket for all endpoints
+
+disabled_limiter = Limiter(
+    default_limits=[CONFIG.security.request_rate_limit],
     headers_enabled=True,
     key_prefix=CONFIG.security.rate_limit_prefix,
     key_func=safe_rate_limit_key,
     retry_after="http-date",
-    in_memory_fallback_enabled=False,  # Fall back to no rate limiting if Redis unavailable
-    enabled=is_rate_limit_enabled,
+    enabled=False,
 )
+
+try:
+    if is_rate_limit_enabled:
+        fides_limiter = Limiter(
+            storage_uri=CONFIG.redis.connection_url_unencoded,
+            application_limits=[
+                CONFIG.security.request_rate_limit
+            ],  # Creates ONE shared bucket for all endpoints
+            headers_enabled=True,
+            key_prefix=CONFIG.security.rate_limit_prefix,
+            key_func=safe_rate_limit_key,
+            retry_after="http-date",
+            in_memory_fallback_enabled=False,  # Fall back to no rate limiting if Redis unavailable
+            enabled=is_rate_limit_enabled,
+        )
+    else:
+        fides_limiter = disabled_limiter
+except Exception as e:
+    logger.warning("Disabling rate limiting due to Redis configuration issue: {}", e)
+    fides_limiter = disabled_limiter
```

Comment:
i think we need to be noisier here because this implies that the intention is to configure rate limiting. that is likely a major concern for the user - and a simple warning message could very easily slip by!

i actually think that if `is_rate_limit_enabled=True`, it probably _does_ make sense to fail startup - we can't really rely on someone finding this in logs, even if it's logged with higher severity. 


```suggestion
    logger.exception("Error instantiating rate limiter: {}", e)
    if is_rate_limit_enabled:
        raise e
```